doc/modules/ROOT/pages/tutorials/fastrp-and-knn.adoc

```diff
@@ -1,6 +1,6 @@
 // DO NOT EDIT - AsciiDoc file generated automatically
 
-= Product recommendations with kNN based on FastRP embeddings
+= Recommendation app (kNN on FastR)
```

Comment:
I don't think it is an app, it's an example. "Recommendation" or "Recommendations" are good in the title, and I want to call out both kNN and FastRP as well, like is retained here (nearly). I also like "embeddings" or even "node embeddings". Not partial to "Product" -- can be omitted. How about

```suggestion
= Recommendations using kNN & FastRP embeddings
```

Reply:
Yes, this one was tricky, but your suggestion is longer than 70. Again we don't need to adapt everything

Reply:
Not much we can do on this, I am afraid.